tests/templates/test_state_preparations.py

```diff
@@ -371,6 +403,100 @@ def circuit(state_vector):
             state_vector = np.array([0, 2, 0, 0])
             circuit(state_vector)
 
+    # fmt: off
```

Comment:
This is to prevent `black` from applying here? Nice! I didn't know about this!

Reply:
its super nice when working with large numpy arrays but want to preserve the 'matrix' shape :D